app/jobs/document_proofing_job.rb

```diff
@@ -100,7 +100,7 @@ def perform(
   def native_camera_ab_test_data(document_capture_session)
     return {} unless IdentityConfig.store.idv_native_camera_a_b_testing_enabled
 
-    ab_test = NativeCameraABTest.new
+    ab_test = Idv::NativeCameraABTest.new
```

Comment:
This is the only one which would need the module scope isn't it? The others are already in the IdV module.

Reply:
Zach made this issue obsolete in #6994